src/cli/load.test.js

```diff
@@ -18,6 +18,8 @@ jest.mock("../plugins/git/loadGitData", () => ({
   loadGitData: jest.fn(),
 }));
 
+export const fakeTimestamp = new Date();
```

Comment:
Having test code that references the actual current date makes me uncomfortable. It's an invitation for flaky tests. It would be better to make the fakeTimestamp a specific fixed date, e.g. new Date(1000)

Reply:
Yep, definitely.

Not sure why we need to export a global `fakeTimestamp` at all. Can’t
each test instantiate its own?

Because date objects are mutable, please don’t use a global constant,
but rather a global function:

```js
const fakeTimestamp = () => new Date(123);
```

Reply:
Definitely! This is the missing piece I was looking for that was giving me trouble with the current unit testing.